### PR TITLE
[DOC] LITE Time classifier metrics

### LINKED ISSUE #2435: [DOC] Type of "metrics" of deep learning classifiers using `keras` could be improved
### Describe the issue linked to the documentation

For example, the `metrics` argument of the `LITETimeCLassifier` (https://www.aeon-toolkit.org/en/stable/api_reference/auto_generated/aeon.classification.deep_learning.LITETimeClassifier.html#aeon.classification.deep_learning.LITETimeClassifier) states:

> metricsstr or list[str], default=”accuracy”
> The evaluation metrics to use during training. If a single string metric is provided, it will be used as the only metric. If a list of metrics are provided, all will be used for evaluation.

However, one can (and has to, if using more complicated metrics) directly use the Keras classes. For example,
```python
        metrics=[
            keras.metrics.AUC(),
            keras.metrics.FBetaScore(average="macro", beta=4.0),
        ],
```

### Suggest a potential alternative/fix

Keras uses the following wording:
> List of metrics to be evaluated by the model during training and testing. Each of this can be a string (name of a built-in function), function or a [keras.metrics.Metric](https://keras.io/api/metrics/base_metric#metric-class) instance.

### PR DESCRIPTION
**Pull request by dschrempf**
State that Keras Metrics class objects (and functions) can be used. Also link to the Keras documentation.

Closes #2435 

#### Any other comments?

Maybe this change is also relevant for other Keras-based (tensorflow-based) estimators?

##### For all contributions
- [ ] I've added myself to the [list of contributors](https://github.com/aeon-toolkit/aeon/blob/main/.all-contributorsrc). Alternatively, you can use the [@all-contributors](https://allcontributors.org/docs/en/bot/usage) bot to do this for you.
- [x] The PR title starts with either [ENH], [MNT], [DOC], [BUG], [REF], [DEP] or [GOV] indicating whether the PR topic is related to enhancement, maintenance, documentation, bugs, refactoring, deprecation or governance.

### CODE REVIEW

**Review by hadifawaz1999 — APPROVED** [review]
LGTM thanks ! So all of our deep learning models should have this change in DOCs, so am happy to get this in now so we can create an issue for it and link this PR as a base example of what should be done, could you create such issue and close the one you made for LITE ?

**Comment by dschrempf** [comment]
I just realized, there was a small typo: if -> If. I can create an issue for the other Keras-based classifiers.

Also, `pyright` (and I guess also `mypy`) do not understand that one can use `keras.metrics.Metric` objects, since `metric` is initialized to "accuray" which is a `str`. However, I didn't want to add a type hint because it seems they are not used much in `aeon`.

**Comment by MatthewMiddlehurst** [comment]
@dschrempf feel free to open an issue. We want to incorporate type hints but theres a lot to go through so they are not required for now. Always welcome though and we can help if they include optional dependencies i.e, `tensorflow`
